javascript/ql/src/semmle/javascript/security/dataflow/CodeInjectionCustomizations.qll

```diff
@@ -51,6 +51,37 @@ module CodeInjection {
     }
   }
 
+  /**
+   * Gets a reference to a `<script />` tag created using `document.createElement`.
+   */
+  private DataFlow::SourceNode scriptTag(DataFlow::TypeTracker t) {
+    t.start() and
+    exists(DataFlow::CallNode call | call = result |
+      call = DOM::documentRef().getAMethodCall("createElement") and
+      call.getArgument(0).mayHaveStringValue("script")
```

Comment:
There has got to be many other ways to construct this kind of element.
Do we already support jQuery's `$("<script></script>")` and friends through other means?

(`$("script")` which selects an existing script element is probably too rare to care about)

Reply:
> Do we already support jQuery's $("<script></script>") and friends through other means?

I don't think so. `.textContent` is only harmful in the context of code-injection and `<script />` tags. 

I can easily find [`<script/>` related jQuery invocations](https://lgtm.com/query/9042076388451521558/).   
But none of them seem to write to the `textContent` of the tag.  
A bunch of them writes to the `src`, but that's another matter.   

So I think it would take too much effort, for too little gain, to support instances where jQuery selects/constructs `<script />` tags.